test/blackbox-tests/test-cases/merlin/allow_approximate_merlin_warn.t/vendor/dune-project

```diff
@@ -0,0 +1,2 @@
+(lang dune 2.8)
```

Comment:
I think for new test we prefer creating these files directly in `run.t`

Reply:
Do we ? I feel like when it doesn't need to change it is nicer this way, but of course I can change it :-)